sdk/servicebus/service-bus/samples/javascript/useProxy.js

```diff
@@ -36,14 +37,18 @@ async function main() {
       // No need to pass the `WebSocket` from "ws" package if you're in the browser
       // in which case the `window.WebSocket` is used by the library.
       webSocket: WebSocket,
-      webSocketConstructorOptions: { agent: proxyAgent }
-    }
+      webSocketConstructorOptions: { agent: proxyAgent },
+    },
+  });
+
+  const sender = sbClient.createSender(queueName);
+
+  console.log(`Sending message using proxy server ${proxyInfo}`);
+
+  await sender.send({
+    body: "sample message",
```

Comment:
```suggestion
    body: "sample message"
```
?

Reply:
That's what prettier is doing so it's probably best not to fight it.

Unless my prettier instance is configured incorrectly?

Reply:
Well...it turns out I was using a different version of prettier. I'll do an `npm run format` and fix this in a PR.